packages/kcms/src/match/utility/uppercase.ts

```diff
@@ -0,0 +1,3 @@
+export const toUpperCase = <T extends string>(str: T): Uppercase<T> => {
+  return str.toUpperCase() as Uppercase<T>;
+};
```

Comment:
- このような関数の名前は動詞がよいでしょう。e.g. `upcase`("up"が動詞、Ruby等)
- ファイル名と関数名は等しくしてください。
- 式のみで書けます。

e.g.
```suggestion
export const upcase = <T extends string>(str: T): Uppercase<T> => str.toUpperCase() as Uppercase<T>;
```